Util/Factory/Query/DoctrineBuilder.php

```diff
@@ -100,8 +101,33 @@ protected function _addSearch(\Doctrine\ORM\QueryBuilder $queryBuilder, array $f
 
                     /** @var DatatableField[] $original_field */
                     $original_field = array_slice($this->fields, $i, 1);
+                    if (isset($filter_fields[$i]) && $filter_fields[$i] instanceof DateTimeFilter)
+                    {
+                        $parts = explode(" - ", $search_param);
+
+                        $start = new \DateTime($parts[0]);
```

Comment:
is it usefull to do something with possible exceptions?

Reply:
I considered this, but rejected it because I'd rather see the error at the earliest stage than somehow catch it here and silence it or some such. This is an error that can only happen with bugs in development of the javascript part. When that's correct, it will not go wrong unless you manually manipulate stuff